extension/src/paymentHandler/payment-handler.js

```diff
@@ -120,35 +119,23 @@ function _validatePaymentRequest(paymentObject, authToken) {
       .validateReference()
       .validateAmountPlanned()
     if (paymentValidator.hasErrors())
-      return {
-        success: false,
-        data: paymentValidator.buildCtpErrorResponse(),
-      }
+      return paymentValidator.buildCtpErrorResponse().errors
```

Comment:
Maybe instead of asking always for `.errors` the `buildCtpErrorResponse()` could be renamed to `getErrors()` which would return an error array right away?

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/656/commits/9bf490870b4ef7dd0d21f0f216d10fbb1bfbc529